tests/e2e/specs/import.spec.js

```diff
@@ -169,8 +183,34 @@ test.describe( 'Feed Import', () => {
 		expect( content ).toContain('<p>');
 		expect( content.split(' ').length ).toBeLessThanOrEqual(30);
 
-		// Enable this test for https://github.com/Codeinwp/feedzy-rss-feeds/pull/946 when the PR is merged.
-		// await page.getByRole('button', { name: 'Featured image' }).click({ force: true });
-		// await expect( page.getByLabel('Edit or replace the image') ).toBeVisible(); // Featured image is added.
+		if( ! await page.getByLabel('Edit or replace the image').isVisible() ) { // Should we open the featured image tab?
+			await page.getByRole('button', { name: 'Featured image' }).click({ force: true });
+		}
+		await page.waitForSelector('.editor-post-featured-image');
```

Comment:
Can we ensure this test is reliable, right now it is skipped as flaky.

Reply:
The interface for checking the featured image in the editor is very flaky on GH Runner.

But since featured posts are a core feature, we can just check how they are saved in the backend. So, I modified the test to see if all the posts created by Feedzy have a featured_media attached.